scripts/main.js

```diff
@@ -0,0 +1,348 @@
+"use strict";
+document.addEventListener("keydown", (event) => move(event.key));
```

Comment:
The 'move' function is called directly in the event listener, but it's not defined above this line. JavaScript hoisting allows function declarations to be lifted, but it's a good practice to define functions before using them to improve readability and maintainability.